### PR TITLE
Add staging environment

### PR DESCRIPTION
**Pull request by enricostano**

### CODE REVIEW

File: inventory/host_vars/example.org/config.yml [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
+
+sys_admins:
+  - name: sys_admin_username
+    ssh_key: "../pub_keys/sys_admin_username.pub"
+    state: present
+
+developers:
+  - name: developer_username
+    ssh_key: "../pub_keys/developer_username.pub"
+    state: present
```
Comment (sauloperez) [inline inventory/host_vars/example.org/config.yml:13]: can we move this to documentation? It leads you to think something different than what's intended.

File: roles/common/tasks/users.yml [hunk 2]
```diff
@@ -8,13 +8,19 @@
...
+- name: Load environment variables in bashrc for `timeoverflow` user
+  blockinfile:
+    path: /home/timeoverflow/.bashrc
+    insertbefore: BOF
+    block: |
+      set -a
+      [ -f /etc/default/timeoverflow ] && . /etc/default/timeoverflow
+      set +a
+  tags:
+    - hola
```
Comment (sauloperez) [inline roles/common/tasks/users.yml]: :warning:

**Review by sauloperez — APPROVED** [review]

File: roles/common/tasks/users.yml (hunk 2 above)
Comment (enricostano) [inline roles/common/tasks/users.yml]: remove this tag

File: inventory/host_vars/example.org/config.yml (hunk 1 above)
Comment (enricostano) [reply inventory/host_vars/example.org/config.yml:13]: Sure, will do in https://github.com/coopdevs/timeoverflow-provisioning/issues/22

**Review by sauloperez — APPROVED** [review]

**Review by danypr92 — APPROVED** [review]

**Pull request merged**